calico-enterprise/network-policy/policy-tiers/tiered-policy.mdx

```diff
@@ -94,7 +102,7 @@ In the example above, tier priorities are as follows:
 
 - **security tier** - is higher priority than platform tier
 - **platform tier** - is higher priority than default tier
-- **default tier** - is always the last tier and cannot be reordered
+- **default tier** - is the lowest priority than the rest
```

Comment:
```suggestion
- **default tier** - is the lowest priority.
```

Previously, is could not be reordered, now it has an order. Can it be reordered? If yes, it would be worth to say that "previously it could not be reordered. It can be now, but we advise against that" or something similar?

Reply:
No, the default tier *must* have the order of 1M. It cannot be reordered. Previously, it had an order of `nil`, and now it is `1M` and it cannot change. Also any  tier with `nil` order will be treated as `1M`.
We will add a release note to warn customers about the change.